src/app/components1/Types/loginCredentials.ts

```diff
@@ -0,0 +1,4 @@
+export class LoginCredential {
```

Comment:
**suggestion:** Follow the [Angular Style Guide](https://angular.io/guide/styleguide#general-naming-guidelines) for naming files, classes and variables.

In this case the file should be named `login-credentials.ts`